gluster-exporter/metric_brick.go

```diff
@@ -577,6 +581,12 @@ func lvmUsage(path string) (stats []LVMStat, thinPoolStats []ThinPoolStat, err e
 
 func brickUtilization(gluster glusterutils.GInterface) error {
 	volumes, err := gluster.VolumeInfo()
+
+	// Reset all vecs to not export stale information
```

Comment:
Let the reset-loop be the first thing/code in the function

Reply:
Otherwise changes look good. Even if there might be a race between retrieving/reading the values and setting the values (which existed in the previous code as well), there should not be any race on setting the values.  No go-routines should compete against each other for setting the values, which is really bad.

Reply:
ok. updated!